infinispan-embedded/query-programmatic-index/src/test/resources/META-INF/services/org.infinispan.search.mapper.mapping.ProgrammaticSearchMappingProvider

```diff
@@ -0,0 +1 @@
+org.infinispan.tutorial.simple.query.programmatic.PersonIndexDefinition
```

Comment:
this file must  be moved to `src/main/resources/META-INF/services/org.infinispan.search.mapper.mapping.ProgrammaticSearchMappingProvider`